build.gradle

```diff
@@ -10,6 +10,8 @@ allprojects {
     apply plugin: 'jacoco'
     apply plugin: 'idea'
     apply plugin: 'maven-publish'
+    // Run `./gradlew htmlDependencyReport` to view the dependency report
```

Comment:
(nit) dead comment?

Reply:
`htmlDependencyReport` is not intuitive to remember  and i i feel this may be useful in future, keeping this comment.